internal/pacseek/setup.go

```diff
@@ -263,6 +264,12 @@ func (ps *UI) setupKeyBindings() {
 			return nil
 		}
 
+		// CTRL+O - opens repo url
+		if event.Key() == tcell.KeyCtrlO {
+		  exec.Command("xdg-open", ps.selectedPackage.URL).Run();
```

Comment:
This segfaults when ps.selectedPackage is nil. There should be check in the if statement.